packages/content/src/Domain/Model/AuthorInterface.php

```diff
@@ -24,4 +24,10 @@ public function setAuthor(?ContactInterface $author): void;
     public function getAuthored(): ?\DateTimeImmutable;
 
     public function setAuthored(?\DateTimeImmutable $authored): void;
+
+    public function getLastModifiedEnabled(): ?bool;
```

Comment:
@alexander-schranz @martinlagler why did we add this here to the `AuthorInterface`?

In 2.6 we had a separate interface for this [LastModifiedBehavior](https://github.com/sulu/sulu/blob/2.6/src/Sulu/Component/Content/Document/Behavior/LastModifiedBehavior.php)

Reply:
As discussed, for BC reasons we coould not add it in 2.6 to other interface. And we thought it makes most sense if I have `authored` date, I also should have the possibility to have a modifyable last modified date.